src/CoreBundle/private/modules/MTTableEditor/MTTableEditor.class.php

```diff
@@ -438,6 +438,8 @@ protected function LoadMessages($bConsumeMessages = true)
     }
 
     /**
+     * @deprecated since 6.3.0 - revision management is no longer supported
```

Comment:
We can also minimize method contents in this class.

Reply:
missed that one. Done